docs/concepts/python-versions.md

```diff
@@ -158,6 +158,27 @@ $ uv python install 3.12.6 --preview  # Does not update `python3.12`
 $ uv python install 3.12.8 --preview  # Updates `python3.12` to point to 3.12.8
 ```
 
+## Upgrading Python patch versions
+
+Managed `python-build-standalone` Python minor versions can be upgraded to the latest patch.
```

Comment:
`python-build-standalone` is an implementation detail, not supposed to be user facing.